dev/io.openliberty.jstl.facade/bnd.bnd

```diff
@@ -19,7 +19,7 @@ Bundle-Description: facade to provide a classloader for our internal jstl classe
 Provide-Capability: com.ibm.ws.jsp.jstl.facade
 
 Import-Package: \
-  jakarta.el;version="[4.0,5.0)", \
+  jakarta.el;version="[4.0,6.0)", \
   jakarta.servlet.jsp;version="[3.0,4.0)", \
```

Comment:
This bundle will be shared between EE9 and EE10?

Reply:
I expect so, unless one of the APIs in the `Import-Package` was removed, but I don't think that's the case?